src/blocks/pacman.rs

```diff
@@ -0,0 +1,53 @@
+use std::time::Duration;
+use std::process::Command;
+
+use block::Block;
+use widgets::text::TextWidget;
+use widget::I3BarWidget;
+
+use serde_json::Value;
+use uuid::Uuid;
+
+
+pub struct Pacman {
+    text: TextWidget,
+    id: String,
+    update_interval: Duration,
+}
+
+impl Pacman {
+    pub fn new(config: Value, theme: Value) -> Pacman {
+        {
+            Pacman {
+                id: Uuid::new_v4().simple().to_string(),
+                update_interval: Duration::new(get_u64_default!(config, "interval", 5), 0),
```

Comment:
This macro: get_u64_default!(config, "interval", 5) actuelly looks for "interval" in the config, and if not specified defaults to 5. The Macro just wraps some nicer error handling to avoid boilerplater, it's in utils.rs.